tests/config/test_data.config

```diff
@@ -145,6 +147,9 @@ params {
                 test2_genome_vcf_gz                            = "${test_data_dir}/genomics/homo_sapiens/illumina/gvcf/test2.genome.vcf.gz"
                 test2_genome_vcf_gz_tbi                        = "${test_data_dir}/genomics/homo_sapiens/illumina/gvcf/test2.genome.vcf.gz.tbi"
                 test2_genome_vcf_idx                           = "${test_data_dir}/genomics/homo_sapiens/illumina/gvcf/test2.genome.vcf.idx"
+
+                test_1_fastq_gz_10x                           = "${test_data_dir}/genomics/homo_sapiens/illumina/10xgenomics/test_1.fastq.gz"
```

Comment:
Ideally, we should rename these files on test-datasets because all of the current keys are based on the exact names of the files 😏 

Maybe `test_1.fastq.gz` -> test.10x_1.fastq.gz